src/Runner.Worker/JobRunner.cs

```diff
@@ -49,6 +49,8 @@ public async Task<TaskResult> RunAsync(AgentJobRequestMessage message, Cancellat
                 VssCredentials jobServerCredential = VssUtil.GetVssCredential(systemConnection);
                 await runServer.ConnectAsync(systemConnection.Url, jobServerCredential);
                 server = runServer;
+
+                _jobServerQueue.Start(message);
```

Comment:
Just verifying that this change should be here as it doesn't look related, but maybe it is?

Reply:
Ah, good eye!  I forgot to remove it.  This will be in next PR where we may skip uploading with `JobServer` but need to start `ResultsServer`.